### DISCUSSION TITLE
Sweeping Volts, Current, and Resistance at  1 Voltage

### DISCUSSION THREAD

**Discussion by Spectyr333** [SpiceSharp/SpiceSharp#177] [Q&A]
Could anyone link me an example or syntax where volts, current, and resistance are run through a sweep a single time at a single voltage? I am having trouble finding anything I can understand. Forgive me for the easy question.

**Comment by svenboulanger** [comment]
Could you please elaborate a little bit more? What do you want to sweep? Just the voltage or do you want to sweep over multiple parameters for a specific circuit? Why both voltage and current? Why a resistance (seems already more specific to your project)?

As a starting point for DC sweeps, you can take a look [here](https://spicesharp.github.io/SpiceSharp/articles/tutorials/dcanalysis.html).

**Reply by Spectyr333** [reply]
I am trying to make a multimeter that measures all of those things. I see references to Current and Voltage and am working on figuring out how to implement them into my needs. But could you show or explain how I could measure resistance between 2 points?
For example: If I have two resistors connected between nodes A and B. Is there a way for Spice# to calculate the resistance between those two nodes?
